Tom/Classes/Configuration.swift

```diff
@@ -5,13 +5,20 @@
 //  Created by Yusuf Demirci on 30.03.21.
 //
 
+/// Configuration parameters for TomView.
 public struct Configuration {
-
+    
+    /// Determines how many lines will be in TomView.
     public let lineCount: Int
+    /// Determines the color of the lines in TomView.
     public let lineColor: UIColor
+    /// Determines spacing behind each line.
     public let lineSpacing: CGFloat
+    /// Determines axis of lines.
```

Comment:
Could you please add the values either? Something like that,
```suggestion
    /// Determines axis of lines, horizontal or vertical.
```